packages/v4/src/content/design-guidelines/components/code-block/code-block.md

```diff
@@ -0,0 +1,34 @@
+---
+id: Code block
+section: components
+---
+
+The **code block** is a component that contains two or more lines of ready-only code. The code in a code block can be copied to the clipboard.
```

Comment:
Change to "a code block" instead of "the code block" for consistency with other guidelines!

Reply:
Can do I'll update that.